lib/Macros/Sources/SwiftMacros/SwiftifyImportMacro.swift

```diff
@@ -353,7 +368,26 @@ struct CountedOrSizedPointerThunkBuilder: PointerBoundsThunkBuilder {
     return ExprSyntax("\(name).\(raw: countName)")
   }
 
-  func getPointerArg() -> ExprSyntax {
+  func peelOptionalType(_ type: TypeSyntax) -> TypeSyntax {
+    if let optType = type.as(OptionalTypeSyntax.self) {
+      return optType.wrappedType
+    }
+    if let impOptType = type.as(ImplicitlyUnwrappedOptionalTypeSyntax.self) {
+      return impOptType.wrappedType
+    }
+    return type
+  }
+
+  func castPointerToTargetType(_ baseAddress: ExprSyntax) throws -> ExprSyntax {
```

Comment:
What is "target type" in this context? Should this be `castPointerToOpaquePointer`?

Reply:
Good point, that's a lot clearer actually